src/devices/src/virtio/block/device.rs

```diff
@@ -146,6 +174,7 @@ pub struct Block {
     pub(crate) partuuid: Option<String>,
     pub(crate) root_device: bool,
     pub(crate) rate_limiter: RateLimiter,
+    pub(crate) ignore_flush: bool,
```

Comment:
Why not use the same `enum CacheType` here as well? Ideally even use it from API all the way down without transformations.

Reply:
Only reason was that `ignore_flush` seemed like a perfect name for the field.

Changed it so that we use `cache_type` as a field in the API request JSON, all the way down to the device.